plugins/modules/postgresql_privs.py

```diff
@@ -77,8 +77,11 @@
   roles:
     description:
     - Comma separated list of role (user/group) names to set permissions for.
-    - The special value C(PUBLIC) can be provided instead to set permissions
-      for the implicitly defined PUBLIC group.
+    - Each role must be in lowercase as internally PostgreSQL store them in lowercase only.
```

Comment:
PostgreSQL stores roles case-sensitive. You can use any random 63 bytes as a role name, as long as it's quoted properly, see https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS.
If an identifier is not quoted, it is converted to lowercase.
Quoting also allows to have roles with confusing names:
```SQL
postgres=# select version();
                                              version                                              
---------------------------------------------------------------------------------------------------
 PostgreSQL 15.1 on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
(1 row)
postgres=# create role public;
ERROR:  role name "public" is reserved
LINE 1: create role public;
                    ^
Time: 1.106 ms
postgres=# create role "Public";
CREATE ROLE
Time: 4.282 ms
```

We should check if quoted role names are supported by this module. It's not a limitation of PostgreSQL.

Reply:
You are right, I forgot to fix docs (the code is already ok).
Done in e408a01a853eb5600b859530e59f802da75c72ac.